### PR TITLE
Refactor/split up get started

### PR DESCRIPTION
**Pull request by mattyg**
**Overview**
- split up `Core Concepts > Get Started` page into multiple separate pages
  - move "hello world" section to `Build > Index`
  - move forum tutorial section to `Build > Forum App Tutorial`
  - move deployment section to `Build > Deployment & Packaging`
  - move unique "Next Steps" content to "References" section
- remove redundant content from `Next Steps` section

### CODE REVIEW

**Comment by pdaoust** [comment]
Seems like a solid move; I'd like to suggest a slight rejiggering:

* keep it in 'Get Started'
* have page 1 of 'Get Started' be the install guide
* Follow with Holo World etc
* Maybe deployment and packaging could be merged with `/get-started/distribute-your-app` and become a proper full-on resource. I wonder if it was out of scope for a getting started guide in the first place.

**Comment by mattyg** [comment]
>   keep it in 'Get Started'

But its not conceptual -- its specific to developers actually building apps. I feel like concepts should be purely for *understanding* holochain not *using* it. When someone is installing the dev tools they've moved solidly into *using* it.

I don't think deployment and packaging belongs in "getting stated" -- I think it deserves it's own page. There's going to be a lot more content needed related to packaging, distribution and updating.

That said I don't have the bandwidth to go back and forth, so I'm happy to let you make the changes you want to this PR and get it merged.

**Comment by pdaoust** [comment]
I appreciate your engaging when you don't have much bandwidth, and I don't want to pull you away too much more. Can I ask one more question though? I'd like clarification on

> But its not conceptual -- its specific to developers actually building apps. I feel like concepts should be purely for understanding holochain not using it. When someone is installing the dev tools they've moved solidly into using it.

So the way I see it, 'Get Started' is all about taking action and starting to use the dev tools (but before you're getting into serious dev work). It isn't currently under Core Concepts, though (which is 100% conceptual) -- maybe that's why the confusion?

> I don't think deployment and packaging belongs in "getting stated" -- I think it deserves it's own page. There's going to be a lot more content needed related to packaging, distribution and updating.

100% agree; not sure why it's there right now.

**Comment by mattyg** [comment]
ah sorry my bad -- yes all your suggestions make sense to me

**Comment by pdaoust** [comment]
This looks good to me now. I would like to get a third set of eyes on it, but I'm not sure who to bring in. @matthewjosef ? @marycamacho ? The tl;dr for this change is that the getting started guide is getting broken up into four pages to make it more digestible.

File: src/pages/get-started/4-packaging-and-distribution.md [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+```
+::: output-block
+
+The packed app is now ready for deployment to a Holochain runtime.
+
+## Runtimes
+
+In the centralized world, deployment is usually achieved by Continuous Integration (CI) automation that builds up code changes and sends them to whatever server or cloud-based platform you're using. In the decentralized world of Holochain, deployment happens when end-users download and run your hApp in the Holochain runtime (although you could certainly use CI to build releases of your hApp).
+
+From the end-user perspective there are currently there are two ways to go about this:
```
Comment (matthme) [inline src/pages/get-started/4-packaging-and-distribution.md:69]: small typo here: "there are currently there are two ways"

File: src/pages/get-started/4-packaging-and-distribution.md [hunk 2]
```diff
@@ -0,0 +1,122 @@
...
+
+1. Download Holochain's official Launcher runtime and install the app from its app store or the filesystem.
+2. Download an your app as its own standalone desktop executable, as they would any other application for their computer.
```
Comment (matthme) [inline src/pages/get-started/4-packaging-and-distribution.md:72]: another small typo here: "Download an your app"

File: src/pages/get-started/4-packaging-and-distribution.md [hunk 3]
```diff
@@ -0,0 +1,122 @@
...
+
+* **Tauri**: See the officially supported [holochain-kangaroo](https://github.com/holochain-apps/holochain-kangaroo) repo.
+* **Electron**: See the officially supported [holochain-kangaroo-electron](https://github.com/holochain-apps/holochain-kangaroo-electron) repo.
+
+Both of these are GitHub template repos with detailed instructions on how to clone them and add in your UI and DNA, as well as build and release commands that will create the cross-platform executables that you can then deliver to your end users. Their supported Holochain version sometimes lags behind the current recommended version, so you may need to manually update them.
```
Comment (matthme) [inline src/pages/get-started/4-packaging-and-distribution.md:89]: Only the tauri one has detailed instructions unfortunately. The electron one has not even a README at the moment :/

**Comment by pdaoust** [comment]
@matthme could you re-review the changes in the last commit, to see if I'm describing your kangaroo repos accurately? Thanks 🙏🏼

File: src/pages/get-started/4-packaging-and-distribution.md (hunk 3 above)
Comment (matthme) [reply src/pages/get-started/4-packaging-and-distribution.md]: Thanks, looking accurate now.

**Review by mattyg — COMMENTED** [review]
I do feel like these pages probably don't belong under "Getting Started", but that can wait until we have the start of a "Build" section:

[Distribute your App](http://localhost:8080/get-started/distribute-your-app/)
[Holochain Upgrade 0.1 → 0.2](http://localhost:8080/get-started/upgrade-holochain/)
[Setup For a Local Event](http://localhost:8080/get-started/at-an-event/)


Otherwise looks good -- so much more legible IMO.

**Comment by pdaoust** [comment]
yeah, those definitely need to move @mattyg -- I was gonna wait until we had a build guide, then create a howtos section.

Although... does 'set up for a local event' feel like it belongs under build? not sure.

**Pull request merged**